Classes/Hooks/Form/FieldInformation/SolrCoreStatus.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * (c) Kitodo. Key to digital objects e.V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo and TYPO3 projects.
+ *
+ * @license GNU General Public License version 3 or later.
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ */
+
+namespace Kitodo\Dlf\Hooks\Form\FieldInformation;
+
+use Kitodo\Dlf\Common\Helper;
+use Kitodo\Dlf\Common\Solr;
+use TYPO3\CMS\Backend\Form\AbstractNode;
+
+/**
+ * FieldInformation renderType for TYPO3 FormEngine
+ *
+ * @author Sebastian Meyer <sebastian.meyer@slub-dresden.de>
+ * @package TYPO3
+ * @subpackage dlf
+ * @access public
+ */
+class SolrCoreStatus extends AbstractNode
+{
+    /**
+     * Shows Solr core status for given 'index_name'
+     *
+     * @access public
+     *
+     * @return array As defined in initializeResultArray() of AbstractNode
+     *               Allowed tags are: "<a><br><br/><div><em><i><p><strong><span><code>"
+     */
+    public function render(): array
+    {
+        $result = $this->initializeResultArray();
+        // Show only when editing existing records.
+        if ($this->data['command'] !== 'new') {
+            $core = $this->data['databaseRow']['index_name'];
+            // Load localization file.
+            $GLOBALS['LANG']->includeLLFile('EXT:dlf/Resources/Private/Language/FlashMessages.xml');
+            // Get Solr instance.
+            $solr = Solr::getInstance($core);
+            if ($solr->ready) {
+                // Get core data.
+                $coreAdminQuery = $solr->service->createCoreAdmin();
+                $action = $coreAdminQuery->createStatus();
+                $action->setCore($core);
+                $coreAdminQuery->setAction($action);
+                $response = $solr->service->coreAdmin($coreAdminQuery)->getStatusResult();
+                $uptimeInSeconds = floor($response->getUptime() / 1000);
```

Comment:
Would be easier to read if adding the amount of days. E.g. DD HH:MM:SS.

I think, this  is what the Solr-Core Admin-Panel writes as "startTime: 16 days ago".

Uptime is better than "Start Time". But The "Star Time" below seems to be the creation time. I'm a little confused.

Reply:
Yeah, I agree that adding the days would be convenient, but it's not easy to calculate: https://stackoverflow.com/questions/12918158/timestamp-to-days-hours-minutes

Reply:
I agree that it would be nice to have the days as well, but it's complicated to calculate. There are so many ways and each has its disadvantages: https://stackoverflow.com/questions/8273804/convert-seconds-into-days-hours-minutes-and-seconds
I think it's just not worth it...